zombienet_tests/0002-pov_recovery.toml

```diff
@@ -33,12 +33,30 @@ cumulus_based = true
   validator = true
   image = "{{COL_IMAGE}}"
   command = "test-parachain"
-  args = ["--reserved-only", "--reserved-nodes {{'charlie'|zombie('multiAddress')}}", "--use-null-consensus", "--disable-block-announcements"]
+  args = ["--reserved-only", "--reserved-nodes {{'charlie'|zombie('multiAddress')}}", "-lcumulus-pov-recovery=debug", "--use-null-consensus", "--disable-block-announcements"]
 
-  # run eve as parachain full node that is only connected to dave
+  # run eve as parachain full node that is only connected to charlie
   [[parachains.collators]]
   name = "eve"
   validator = false
   image = "{{COL_IMAGE}}"
   command = "test-parachain"
-  args = ["--reserved-only", "--reserved-nodes {{'charlie'|zombie('multiAddress')}}", "--use-null-consensus", "--disable-block-announcements"]
+  args = ["--reserved-only", "--reserved-nodes {{'charlie'|zombie('multiAddress')}}", "-lcumulus-pov-recovery=debug", "--use-null-consensus", "--disable-block-announcements"]
+
+  # Run one as parachain collator and two as parachain full node, connected via RPC relay chain node
+  #
+  # They will need to recover the pov blocks through availability recovery.
+  [[parachains.collators]]
+  name = "one"
+  validator = true
+  image = "{{COL_IMAGE}}"
+  command = "test-parachain"
+  args = ["--reserved-only", "--reserved-nodes {{'charlie'|zombie('multiAddress')}}", "-lcumulus-pov-recovery=debug", "--use-null-consensus", "--disable-block-announcements", "--relay-chain-rpc-url {{'alice'|zombie('wsUri')}}", "-- --bootnodes {{'alice'|zombie('multiAddress')}}"]
+
+  # run two as parachain full node that is only connected to dave
+  [[parachains.collators]]
+  name = "two"
+  validator = false
+  image = "{{COL_IMAGE}}"
+  command = "test-parachain"
+  args = ["--reserved-only", "--reserved-nodes {{'charlie'|zombie('multiAddress')}}", "-lcumulus-pov-recovery=debug","--use-null-consensus", "--disable-block-announcements", "--relay-chain-rpc-url {{'bob'|zombie('wsUri')}}"]
```

Comment:
Wouldn't it be slightly more real-world if these parachain nodes would connect to non-validator relay nodes as their relay chain RPC nodes?